scripts/bootstrap_local

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+WORKSPACE="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+source scripts/common
+source scripts/bootstrap_fun
+
+travis_fold_start() { :; }
+travis_fold_end() { :; }
+
+IVY2_DIR="$HOME/.ivy2"
+# The quick stage `publish`es to `integrationRepoUrl`. This makes it into a `publishLocal`.
+integrationRepoUrl="file://$IVY2_DIR/local"
```

Comment:
this doesn't work as intended, as it publishes there in maven format, but should be ivy format. need do actually use `publishLocal`.